data/json/npcs/refugee_center/surface_visitors/NPC_arsonist.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "type": "item_group",
+    "id": "NC_ARSONIST_storage",
```

Comment:
It doesn't seem like you're adding a reference to this new group anywhere.  Does it just magically get used based on the name?

Reply:
Yeah, I dunno exactly how it works but I saw the golf bag show up in-game.

Reply:
It's in npc.cpp - look at `random_item_from` and below.